From 7fad1125220bb72237c4381c7d8bd09c78bffe5a Mon Sep 17 00:00:00 2001
From: Shreenidhi Shedi <sshedi@vmware.com>
Date: Tue, 22 Mar 2022 20:28:04 +0530
Subject: [PATCH] give watch permission to agetty

This fixes the below avc denial issue:

Mar 22 14:55:05 photon-machine audit[675]: AVC avc:  denied  { watch } for  pid=675 comm="(agetty)" path="/dev/tty1" dev="devtmpfs" ino=20 scontext=system_u:system_r:init_t:s0 tcontext=system_u:object_r:tty_device_t:s0 tclass=chr_file permissive=0

Signed-off-by: Shreenidhi Shedi <sshedi@vmware.com>
---
 policy/modules/system/init.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/system/init.te b/policy/modules/system/init.te
index 9ed2ba0..aceb3f1 100644
--- a/policy/modules/system/init.te
+++ b/policy/modules/system/init.te
@@ -373,6 +373,7 @@ term_use_usb_ttys(init_t)
 term_use_all_ptys(init_t)
 term_setattr_all_ptys(init_t)
 term_use_virtio_console(init_t)
+term_watch_unallocated_ttys(init_t)
 term_watch_reads_unallocated_ttys(init_t)
 
 # Run init scripts.
-- 
2.25.1

